playbooks/openshift-checks/pre-install.yml

```diff
@@ -0,0 +1,15 @@
+---
+- include: ../openshift-cluster/initialize_groups.yml
+
+- hosts: OSEv3
+  # Temporarily reverting to OSEv3 until group standardization is complete
+  name: run OpenShift pre-install checks
+  roles:
+    - openshift_health_checker
+  post_tasks:
+    # NOTE: we need to use the old "action: name" syntax until
+    # https://github.com/ansible/ansible/issues/20513 is fixed.
```

Comment:
https://github.com/ansible/ansible/issues/20513 is stale, we can probably just get rid of this comment as well (here and in the other playbook).
The only purpose it serves is to stop someone from "fixing" the YAML to use the other syntax and breaking the playbook. Is it a worth cause?